scripts/upgrade/sabnzbd.sh

```diff
@@ -47,13 +47,15 @@ if dpkg --compare-versions ${localversion} lt ${latestversion}; then
 
         echo_progress_done
     fi
-    cd /opt/
     echo_progress_start "Downloading latest source"
-    wget -q -O sabnzbd.tar.gz $latest
-    rm -rf sabnzbd/*
-    sudo -u ${user} bash -c "tar xzf sabnzbd.tar.gz --strip-components=1 -C /opt/sabnzbd" >> "$log" 2>&1
+    wget -q -O /tmp/sabnzbd.tar.gz "$latest"
+    rm -rf /opt/sabnzbd/*
+    if [[ ! -d /opt/sabnzbd ]]; then
+        mkdir -p /opt/sabnzbd
+    fi
```

Comment:
```suggestion
    rm -rf /opt/sabnzbd/
    mkdir -p /opt/sabnzbd
```
maybe a little simpler? not sure whether there's some edge-case this is trying to catch